web/static/styles.css

```
@@ -0,0 +1,1558 @@
+/* birdnet-go-remote-mic Concept B: Precision Field Audio Instrument
+   Design tokens, responsive layouts, and tactical component styling. */
+
+/* Self-hosted fonts (latin subset). Embedded in the binary; no external CDN.
+   Inter for UI text, JetBrains Mono for readouts. */
+@font-face { font-family: "Inter"; font-style: normal; font-weight: 400; font-display: swap; src: url("fonts/inter-400.woff2") format("woff2"); }
```

Stylelint (17.14.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)